tests/plan/show/test.sh

```diff
@@ -71,7 +71,7 @@ rlJournalStart
         rlRun -s "tmt plan show $plan -vvv"
         dump_fmf_id_block $rlRun_LOG > $show_tmp
         rlRun "cat $show_tmp"
-        rlAssertNotGrep "ref:" $show_tmp
+        rlAssertGrep "ref:" $show_tmp
```

Comment:
Seems that `The ref should not be shown when under the default branch` check is now irrelevant. Shall we drop it  as a whole?

Reply:
Hm, true, it does not make sense anymore, it will be shown always.

Reply:
Refactored slightly in 01df02ff. I still see some value in checking for those fields.

Reply:
Ack, makes sense.